src/components/SearchBar/SearchBar.spec.js

```diff
@@ -0,0 +1,22 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
+import SearchBar from './index';
+
+describe('SearchBar component', () => {
+	it('should render children', () => {
+		const handleChange = jest.fn();
+
+		render(<SearchBar searchChallenges={handleChange} searchInput="the" />);
```

Comment:
Since you have a callback function as prop, I would suggest to add another test, which tests that the function is called.

Reply:
I would like to test it, but I don't know how to test an onClick function.

Reply:
You can use the `userEvent` :) with `userEvent.type(...)` https://testing-library.com/docs/user-event/utility#type